portal-bridge/src/cli.rs

```diff
@@ -252,6 +252,11 @@ impl FromStr for ClientType {
 
 impl ClientType {
     pub fn build_handle(&self, bridge_config: &BridgeConfig) -> anyhow::Result<Child> {
+        if !bridge_config.executable_path.is_file() {
+            return Err(anyhow::anyhow!(
+                "Invalid executable path: exectuable doesn't exist"
```

Comment:
*nitpick* import `anyhow!` at top of file. Also, I think it'd be useful to add the path that failed the lookup to the error message